tests/test_parquet_to_root.py

```diff
@@ -9,16 +11,18 @@
 skhep_testdata = pytest.importorskip("skhep_testdata")
 
 
-def test_hepdata():
+def test_hepdata(tmp_path):
     arrays = uproot.open(skhep_testdata.data_path("uproot-hepdata-example.root"))[
         "ntuple;1"
     ].arrays()
 
-    ak.to_parquet(arrays, "uproot-hepdata-example.parquet")
+    ak.to_parquet(arrays, Path(tmp_path) / "uproot-hepdata-example.parquet")
```

Comment:
There's no need to wrap `tmp_path` again here, as it is explicitly a `pathlib.Path` object. so all these can be just `tmp_path` instead of `Path(tmp_path)`.